src/Compilers/CSharp/Test/Emit2/Semantics/RefReadonlyParameterTests.cs

```diff
@@ -4606,6 +4734,30 @@ static void Main()
             Diagnostic(ErrorCode.WRN_OverridingDifferentRefness, "M2").WithArguments("in int x", "ref readonly int x").WithLocation(9, 12));
     }
 
+    [Fact]
+    public void Implementation_RefReadonly_In_Close()
```

Comment:
>Implementation_RefReadonly_In_Close [](http://example.com/codeflow?start=16&length=35)

What is the goal of the test? Should we test success scenario instead by making the methods in ```C``` public?

Reply:
No, the point is to test `MemberSignatureComparer.CSharpCloseImplicitImplementationComparer`